packages/client/src/widgets/SignInForm/ui.tsx

```diff
@@ -0,0 +1,44 @@
+import { Form, FormInput } from '@/shared/ui/Form'
+import { Button } from '@/shared/ui/Button'
+import { ROUTES } from '@/app/router/config'
+import { useNavigate } from 'react-router-dom'
+import authServices from '@/shared/services/authServices'
+import type { TSignInSent } from '@/shared/services/authServices/types'
+
+const SignInForm = () => {
```

Comment:
Проще сделать export рядом:
```suggestion
export const SignInForm = () => {
```

Reply:
Я всегда придерживался принципа "1 файл = 1 по экспортируемая по умолчанию сущность". Здесь с ребятами решили делать иначе, но принцип "одна главная сущность экспортируется в конце файла" кажется мне более удобным, чем экспортировать сущность сразу после её объявления.

Reply:
> но принцип "одна главная сущность экспортируется в конце файла"

В таком случае стоит сделать его дефолтным
```ts
export default SignInForm;
```

Нет смысла написать
```ts
export {
    SignInForm
}
```
так как это равнозначно то чему я написал выше.

Такой стиль был в requireJS, где обычные модули не были.

Reply:
Я понимаю. Но по причине наличия специфичных ре-экспортов в соседнем индекс файле решили делать из юай-файла импорт именованный. 

Визуально удобнее анализировать файл и не видеть "шума" (а-ля export, длинных дженерик-типов, деструктуризации всего и вся прямо в аргументе и т. д.)

Экспорт в конце файла удобен тем, что можно пролистать файл сразу до конца и увидеть, что из файла "отдаётся" одна сущность - компонент-функция, пусть и именованная.

Reply:
Это не настолько холиварная тема, сделайте как вам удобнее, up to u.